### ISSUE TITLE
Python template app crashes on 16.04

### ISSUE THREAD

**Issue by jonnius** [bhdouglass/clickable#84]
How to reproduce:
- Run `clickable init`
- Choose _[3] python-cmake - Python/QML App (built using CMake)_
- Enter directory and run `clickable --xenial` (the app builds and installs, but crashes on start)
- Run `clickable logs` and you'll get:
```
Loading module: 'libubuntu_application_api_touch_mirclient.so.3.0.0'
Could not find platform independent libraries <prefix>
Could not find platform dependent libraries <exec_prefix>
Consider setting $PYTHONHOME to <prefix>[:<exec_prefix>]
Fatal Python error: Py_Initialize: Unable to get the locale encoding
ImportError: No module named 'encodings'
```

**Comment by bhdouglass** [comment]
The python template isn't setup for xenial yet. You'll probably need to remove the embedded python libs and use the ones provided by the system (since they come with the image now).

**Comment by bhdouglass** [comment]
This has been fixed in v5.0.0 (releasing soon!).
